User belongs to experiment group 'pythonaacf'
User belongs to experiment group 'pythonJediLSP'
User belongs to experiment group 'pythonDiscoveryModuleWithoutWatcher'
User belongs to experiment group 'pythonTensorboardExperiment'
User belongs to experiment group 'PythonPyTorchProfiler'
User belongs to experiment group 'pythonDeprecatePythonPath'
User belongs to experiment group 'pythonSortEnvs'
Python interpreter path: ./venv/bin/python
Starting Jedi language server.
Error 2021-08-25 22:32:32: Failed to check if file needs to be fixed [EntryNotFound (FileSystemError): Unable to read file '/home/neb/Dropbox/pie/flac_fixer/.vscode/settings.json' (Error: Unable to resolve non-existing file '/home/neb/Dropbox/pie/flac_fixer/.vscode/settings.json')
	at _handleError (/opt/visual-studio-code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:85:158459)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async y.readText (/home/neb/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:9:320331)
	at async p.doesFileNeedToBeFixed (/home/neb/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:59:586718)
	at async /home/neb/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:59:585845
	at async Promise.all (index 1)
	at async p.getFilesToBeFixed (/home/neb/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:59:585791)
	at async p.updateTestSettings (/home/neb/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:59:585418)] {
  code: 'FileNotFound'
}
> conda --version
Error 2021-08-26 14:42:00: Error while running worker[Worker 1]. undefined
Error 2021-08-26 14:42:00: Error while running worker[Worker 2]. undefined
